soc/microchip/mec/Kconfig

```diff
@@ -216,7 +210,17 @@ config MCHP_HEADER_VERBOSE_OUTPUT
 
 endif # MCHP_MEC_UNSIGNED_HEADER
 
-# Select SoC Part No. and configuration options
-source "soc/soc_legacy/arm/microchip_mec/*/Kconfig.soc"
+config SOC_MEC172X_PROC_CLK_DIV
```

Comment:
This one is used by mec15xx drivers as well as it turns out, should probably be renamed but I just moved it up the hierarchy in the interest of not changing the matching usage in drivers/

Reply:
Falls on the maintainers responsibility to fix

Reply:
yeah fine by me... I can also rename it here if you prefer, whatever, no difference but this change is needed for the boards to build